js/widgets/pagecontainer.js

```diff
@@ -939,7 +939,7 @@ define( [
 				historyDir, pageTitle, isDialog,
 				alreadyThere, newPageTitle,
 				params,	cssTransitionDeferred,
-				beforeTransition;
+				beforeTransition, focusElement;
```

Comment:
I'm not sure what you mean here. Variables that are not being initialized are being listed together at the beginning of the declaration, subsequent lines are indented, the new variable uses complete words. Where does this deviate from the style guide?